homeassistant/components/syncthru/sensor.py

```diff
@@ -173,8 +186,25 @@ async def async_update(self):
                 self.syncthru.url,
             )
             self._active = False
-        self._state = SYNCTHRU_STATE_HUMAN[self.syncthru.device_status()]
-        self._attributes = {"display_text": self.syncthru.device_status_details()}
+        self._state = self.syncthru.is_online()
+        self._attributes = {
+            "detailed_state": SYNCTHRU_STATE_HUMAN[self.syncthru.device_status()],
```

Comment:
Can we instead create separate sensors for the detailed state and disable that sensor by default?

Reply:
Is disabling via setting "enabled" to False correct? Or is there a different kind of disabling?

Reply:
We need to use this:
https://developers.home-assistant.io/docs/entity_registry_disabled_by#integrations-setting-default-value-of-disabled_by-for-new-entity-registry-entries

Reply:
I am wondering: is the "async_update" method called for disabled entities? because updates in this sensor are (as of now) the only "real" updates, all other sensors simply rely on this sensor to fetch the new data.

Reply:
No, the default polling will be turned off for disabled entities.

Reply:
I see. Then the polling should move outside of this entity - maybe into an independent async tracking task?

Reply:
Use a data coordinator instead:
https://developers.home-assistant.io/docs/integration_fetching_data#coordinated-single-api-poll-for-data-for-all-entities